user_templates/overrides.cfg

```diff
@@ -24,26 +24,43 @@
 #-------------------------#
 
 ## The START_PRINT sequence is modular and fully customizable. A default START_PRINT sequence is auto-populated based on
-## your probe choice (TAP, Dockable, Inductive), but if for some reasons you still want to modify it, please uncomment the
-## following `variable_startprint_actions` definition. You can use any number of action or even duplicate some actions if needed.
-## Available actions: "bed_soak", "chamber_soak", "extruder_heating", "tilt_calib", "z_offset", "bedmesh", "purge", "clean", "primeline"
+## your probe choice (TAP, Dockable, Inductive), but if for some reasons you still want to modify it, please uncomment the following 3
+## lines to define a new `variable_startprint_actions`. You can use any number of action or even duplicate some actions if needed.
+## Available actions: "bed_soak", "extruder_preheating", "chamber_soak", "extruder_heating", "tilt_calib", "z_offset", "bedmesh", "purge", "clean", "primeline"
 ##
+# [gcode_macro _USER_VARIABLES]
 # variable_startprint_actions: "action1", "action2", ...
+# gcode: # do not remove this line
 
 
 #-------------------------#
 #   EXTRUDER/BED PIDs     #
 #-------------------------#
 
-[extruder]
-control: pid
-pid_kp: 26.015
-pid_ki: 1.770
-pid_kd: 95.603
-
-[heater_bed]
-control: pid
-pid_kp: 38.771
-pid_ki: 1.746
-pid_kd: 215.177
+## Here is an example of some overrides for the PIDs. I put them here as almost everyone will use them,
+## so just uncomment the following lines and change the values according to your own machine
+# [extruder]
+# control: pid
+# pid_kp: 26.015
+# pid_ki: 1.770
+# pid_kd: 95.603
+
+# [heater_bed]
+# control: pid
+# pid_kp: 38.771
+# pid_ki: 1.746
+# pid_kd: 215.177
```

Comment:
i recommend adding some info regarding how to obtain the values and that the save_config doesn't work as this section is not placed in printer.cfg (then save config would work)

Reply:
I indeed want to add a special comment somewhere to say that SAVE_CONFIG do not work with Klippain.

(It's easier to do that instead of adding PID to printer.cfg and/or some of the other things as then we would need to explain: this works using SAVE_CONFIG, but this does not, etc... and confuse people to "why". So I just prefer to keep the files clean and easy and say "SAVE_CONFIG" doesn't work in Klippain and that's it)